### ISSUE TITLE
Does Meshroom reconstruction contain the size infomation of the model?

### ISSUE THREAD

**Issue by citystrawman** [alicevision/Meshroom#2614]
**Describe the problem**
My target is to reconstruct the model that contains the size infomation. I do not require the size to be accurate, about right is acceptable.

Taking the official tree example, I reconstructed the model and measured it in meshlab, the size is shown as follows:
![image](https://github.com/user-attachments/assets/3b2a3497-35e4-43cb-ad02-da0cf2e9fcd2)

So here's my question: does the measured 2.62525 correctly reflect the tree's size infomatioin? **If not, then, is there a quick and solid way to get the size infomation?**  What I can imagine is to add a marker (which has a known size) when taking the photo, after the model is reconstructed, rescale the model such that the marker's size is its actual size. I hope if there should be a better and more effective way. I appreciate if anyone could help me solve this issue. Thank you!

**Comment by msanta** [comment]
No, what you measured is not the actual size of the object. As you have stated you can add some marker/object of a known size and then scale the mesh. Or you can use two CCTAGs with a known distance between them which will give you a more accurate result (see https://github.com/alicevision/Meshroom/wiki/CCTAG-scaling).

**Comment by natowi** [comment]
...or measure the distance of to points in the model you know the real dimensions from, then you can calculate the scaling factor you can use in the manual scaling node settings.

**Comment by citystrawman** [comment]
> No, what you measured is not the actual size of the object. As you have stated you can add some marker/object of a known size and then scale the mesh. Or you can use two CCTAGs with a known distance between them which will give you a more accurate result (see https://github.com/alicevision/Meshroom/wiki/CCTAG-scaling).

Thank you. BTW is it possible to use CCTAGs by commandline tool instead of the GUI? I suppose that if the software could recognize CCTAGs' ID, then I can preset these CCTAGs' coordinates in a file and by reading the file at sfmTransform node, the commandline tool could scale and transform the model to its actual position/size.

**Comment by natowi** [comment]
https://github.com/alicevision/Meshroom/issues/2359
